### PR TITLE
Feature: l15 config provided via flag

### PR DESCRIPTION
**Pull request by blazejkrzak**
This PR allows to provide flag for config `-l15` to reduce the complexity behind config propagation

### CODE REVIEW

**Comment by blazejkrzak** [comment]
This is built binary from my pc UBUNTU_18.04
https://swarm-gateways.net/bzz:/90a01df373acd86cc56f040d5f2b24bd58e63022976b82d53d5ac50e31e944a8/beacon-chain

File: shared/featureconfig/flags.go [hunk 1]
```diff
@@ -21,6 +21,10 @@ var (
 	PraterTestnet = &cli.BoolFlag{
 		Name:  "prater",
 		Usage: "Run Prysm configured for the Prater test network",
+	} // PraterTestnet flag for the multiclient eth2 testnet.
```
Comment (mxmar) [inline shared/featureconfig/flags.go]: I think this should looks like `L15TestNet flag for the LUKSO eth2 testnet.` here.

**Review by mxmar — CHANGES_REQUESTED** [review]

File: shared/featureconfig/flags.go (hunk 1 above)
Comment (blazejkrzak) [reply shared/featureconfig/flags.go]: this line is ok, its for a prater testnet
Screenshot from 2021-09-23 10-02-08

Comment (blazejkrzak) [reply shared/featureconfig/flags.go]: its done twice, I'll erase it

**Comment by blazejkrzak** [comment]
whats the status of it? It was taken or it is still waiting? @atif-konasl @meta-bot

**Comment by atif-konasl** [comment]
It's already merged with our v1.4.4 version. so i am closing this.

**Pull request closed without merging**